gravity.sh

```diff
@@ -497,13 +498,23 @@ gravity_DownloadBlocklists() {
   # Use compression to reduce the amount of data that is transferred
   # between the Pi-hole and the ad list provider. Use this feature
   # only if it is supported by the locally available version of curl
-  if curl -V | grep -q "Features:.* libz"; then
+  curlVersion=$(curl -V)
```

Comment:
`curlVersion` is not needed anymore, as we use a different command now to check for `etag` support.